trampoline.html

```diff
@@ -0,0 +1,24 @@
+---
+layout: null
+---
+<!DOCTYPE html>
+<html lang="fr">
+    <head><title>Trampoline</title></head>
+    <body>
+        <script src="https://cdn.polyfill.io/v2/polyfill.js?features=default,fetch"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/URI.js/1.19.0/URI.js"></script>
+        <script>
+          let params = new URI(window.location.href).query(true)
+          let what = params.what
+          let where = params.where
+          let templateUrl = "https://raw.githubusercontent.com/betagouv/beta.gouv.fr/templates/"+what+".md"
```

Comment:
Quitte à faire du ES6 : `https://raw.githubusercontent.com/betagouv/beta.gouv.fr/templates/${what}.md"` ?

Reply:
Ah c'est ça la syntaxe, j'avais essayé mais pas retrouvé comment les écrire :)

Il faudrait un polyfill sur IE11 si on veut être cohérent avec le polyfill pour le fetch, mais je ne sais pas dans quelle mesure on trouve facilement des polyfill pour les fonctionnalités syntaxiques; peut-être qu'il serait plus raisonnable dans ce cas de revenir à la syntaxe ES5 (juste supprimer les arrow functions, sauf si je me suis laissé aller par inadvertence à utiliser d'autres trucs de ES6).

WDYT ?

Reply:
Je pense que `let` c'est aussi de l'ES6.

Perso je pense que IE11 OSEF, les gens qui vont utiliser le trampoline ont un navigateur moderne, et s'ils n'en ont pas, je pense qu'on veut les forcer à passer sur une version récente. Et rajouter les polyfill ça rajoute des dépendances (certes fiables, mais bon, c'est pour le principe).

Donc je suis d'avis qu'on peut faire du ES6 sans polyfill, mais @Flightan est visiblement d'un avis différent du mien :)

Reply:
@Flightan Tu voterais "ES5 and polyfill all the things" ou "ES6 and OSEF" ?

Reply:
@gregoirenovel On peut utiliser let dans IE11 sans polyfill, mais il faut le polyfiller dans d'autres browsers, go figure. https://caniuse.com/#feat=let

Reply:
ES6 semble supporté par la majorité des navigateurs desktop à jour c'est pourquoi (contexte où il va être executé), donc à mon avis on peut vraiment faire l'impasse sur tout fallback :)